packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -32,6 +32,7 @@ import {
   getTypeOfAxis,
   tooltipOfXAxislabels,
 } from '../../utilities/index';
+import { formatPrefix as d3FormatPrefix } from 'd3-format';
```

Comment:
What is the behavior if the no of bars exceed the available width of the canvas

Reply:
The bar width readjusts to maintain 2:1 spacing